index.js

```diff
@@ -141,6 +142,11 @@ function isbundle(zipfile, callback) {
       return callback(null, true);
     }
 
+    //bundle: converted from gpx/kml (1 to n geojsons) and has metadata.json file and archived gpx/kml file
+    if (has_archivedfile && has_metadata && unique_extensions.indexOf('.geojson') > -1) {
```

Comment:
Thanks for pointing this out. The second check is redundant since having an archived file is optional. I've removed this check.